solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -152,6 +152,14 @@ contract RandomBeacon is Ownable {
     ///         operator affected.
     uint256 public relayEntryTimeoutNotificationRewardMultiplier;
 
+    /// @notice Percentage of the DKG malicious result notification reward
```

Comment:
Actually, this should say "Percentage of the staking contract malicious behavior notification reward...", same for `relayEntryTimeoutNotificationRewardMultiplier`. Can we fix it? 🙏🏻

Reply:
Sure: https://github.com/keep-network/keep-core/pull/2695/commits/0d6f0ab25225ac5f2407f3a0d07b2b922fd33c89